app/experimenter/nimbus-ui/README.md

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Let's add some docs (even a one-liner, or just copy directly from fxa-settings) about what Storybook is and how to start it.